### PR TITLE
Fix partner address to be clearable

### PR DESCRIPTION
**Pull request by daniel-leod**
API PR: https://github.com/SeedCompany/cord-api-v3/pull/2468

### CODE REVIEW

File: src/scenes/Partners/Edit/EditPartner.tsx [hunk 1]
```diff
@@ -190,6 +191,7 @@ export const EditPartner = ({
                 input: {
                   partner: {
                     ...rest,
+                    address: address?.trim().length ? address : null,
```
Comment (CarsonF) [inline src/scenes/Partners/Edit/EditPartner.tsx:194]: I think with https://github.com/SeedCompany/cord-api-v3/pull/2469 the field will be cleared now without this. Can you try?

Comment (daniel-leod) [reply src/scenes/Partners/Edit/EditPartner.tsx:194]: I tried yesterday but I think it's still needed. Let me double-check.

Comment (daniel-leod) [reply src/scenes/Partners/Edit/EditPartner.tsx:194]: Ok, just tried and to be able to clear the partner address it's needed to submit a blank space `' '` or more. That doesn't feel very intuitive to me.

**Review by CarsonF — APPROVED** [review]
Thanks @daniel-leod!

**Pull request merged**